src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/UpToDate/UpToDateCheckImplicitConfiguredInputDataSource.cs

```diff
@@ -63,9 +62,8 @@ protected override IDisposable LinkExternalInput(ITargetBlock<IProjectVersionedV
 
             IProjectValueDataSource<IProjectSubscriptionUpdate> source1 = _configuredProject.Services.ProjectSubscription.JointRuleSource;
             IProjectValueDataSource<IProjectSubscriptionUpdate> source2 = _configuredProject.Services.ProjectSubscription.SourceItemsRuleSource;
-            IProjectValueDataSource<IProjectSnapshot> source3 = _configuredProject.Services.ProjectSubscription.ProjectSource;
```

Comment:
We no longer synclink on `ProjectSource`, which will cut down the number of updates we handle, reducing GC pressure.